api/db-cdr/generate-cdr/run-achilles-queries.sh

```diff
@@ -1116,11 +1118,27 @@ Count(*) as count_value,0 as source_count_value
 FROM \`${BQ_PROJECT}.${BQ_DATASET}.observation\` o join \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.survey_question_map\` sq
 On o.observation_source_concept_id=sq.question_concept_id
 join \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.survey_module\` sm on sq.survey_concept_id = sm.concept_id
-Where (o.observation_source_concept_id > 0 and o.value_as_number is not null)
+Where (o.observation_source_concept_id > 0 and o.value_as_number >= 0 and o.observation_source_concept_id not in (${toBinSurveyQuestions[@]}) )
 and o.value_as_number > 0
 Group by o.observation_source_concept_id,o.value_as_number,sm.concept_id,sq.id
 order by sq.id asc"
 
+# Bin and set the survey answer count for the survey question answers that has free text value
+bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
+"insert into \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.achilles_results\`
+(id,analysis_id,stratum_1,stratum_2,stratum_4,stratum_5,count_value,source_count_value)
+SELECT 0 as id, 3110 as analysis_id,CAST(sm.concept_id as string) as stratum_1,CAST(o.observation_source_concept_id as string) as stratum_2,
+CAST(floor(o.value_as_number/10)*10 as string) as stratum_4,cast(sq.id as string) stratum_5,
```

Comment:
How does binning by 10 relate to the `--bin-size` flag which is passed to `make-public-bq-data.sh` ? (where 20 is generally used)

Reply:
This was not related to the bin size at all. I was just placing them in 10 sized buckets for display based on the values.